backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/migration/Migration.java

```diff
@@ -20,13 +20,24 @@
 package org.apache.james.backends.cassandra.migration;
 
 import org.apache.james.task.Task;
+import org.apache.james.task.TaskType;
 
 public interface Migration {
 
     void apply() throws InterruptedException;
 
     default Task asTask() {
-        return this::runTask;
+        return new Task() {
```

Comment:
I don't understand this change: what is going to happen when we'll put this task into the task manager?

Reply:
It's the migration subtask that you serialized. If we understanded it well at  the time they are never Handled by the taskmanager.

Reply:
they are executed in `MigrationTask`